_config.yml

```diff
@@ -16,7 +16,7 @@ foreman_latest_releases:
   - minmaj: '3.8'
     patch: '0'
   - minmaj: '3.7'
-    patch: '0'
+    patch: '1'
```

Comment:
With 3.9 now out, there is no '3.7' section.  This will need to be removed.  https://github.com/theforeman/theforeman.org/blob/gh-pages/_config.yml#L15